src/makeCalculator.js

```diff
@@ -37,7 +37,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    operate(operation, number) {
+      this.result = operation(this.result, number);
```

Comment:
Some students call the argument of "operate" just `callback`, it's not really clear to me, but `operation` it's really clear! GJ 🔥